tests/unit_tests/distributed/fsdp/test_mfsdp_fully_shard.py

```diff
@@ -159,10 +159,16 @@ def build_toy_model_and_optimizer(model_type: str, init_model_with_meta_device:
                 model_dim=DIM_SIZE, num_heads=2, num_layers=NUM_LAYERS, output_dim=DIM_SIZE
             )
             fsdp_unit_modules = [te.pytorch.TransformerLayer]
-        toy_adam = Adam(params=toy_model.parameters(), lr=0.01)
 
     # Return the toy model, optimizer, and FSDP unit modules.
-    return toy_model, toy_adam, fsdp_unit_modules
+    return toy_model, fsdp_unit_modules
+
+
+def build_toy_optimizer(model: torch.nn.Module):
```

Comment:
Can we remove `build_toy_optimizer` funciton? Initializing `Adam` inline seems more concise.